Angular - Frontend/src/app/Services/download-file.service.ts

```diff
@@ -0,0 +1,25 @@
+import {Injectable} from '@angular/core';
+import {HttpClient, HttpEvent, HttpRequest} from '@angular/common/http';
+import {Observable} from 'rxjs';
+import {debug} from 'util';
+import {map} from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DownloadFileService {
+  private downloadApi: string;
+
+  constructor(private httpClient: HttpClient) {
+    this.downloadApi = 'http://localhost:8081/downloadFile/';
```

Comment:
this URL should be placed in enviroment.ts

Reply:
done